.github/workflows/test.yml

```diff
@@ -64,12 +64,12 @@ jobs:
       # Build and run test container
       - name: Run all tests
         run: |
-          gcloud compute ssh "zebrad-tests-$BRANCH_NAME-$SHORT_SHA" --zone "$ZONE" --command \
+          gcloud compute ssh "zebrad-tests-$BRANCH_NAME-$SHORT_SHA" --ssh-flag="-o ServerAliveInterval=5" --zone "$ZONE" --command \
```

Comment:
Do you think the ssh flag is necessary? I'd like to merge with the "-t" changes to calling Docker first perhaps

Reply:
I think it's not necessary, it may just help to check what is getting stuck (otherwise it will disconnect after 10 min). But yes, let's try with only -t first. Removed the ssh flag in 10a9c700

Reply:
I'm beginning to think you are right to include `ServerAliveInterval=5`, the test job I ran is still going after an hour, whereas the ones with the flag wrapped up in ~40 minutes: https://github.com/ZcashFoundation/zebra/runs/3343158038?check_suite_focus=true

Reply:
Weird! Do you want me to re-add it? Feel free to commit in this PR if that's faster too

Reply:
Re-added, testing: https://github.com/ZcashFoundation/zebra/runs/3344568057?check_suite_focus=true

Reply:
It worked!